pkg/ddc/alluxio/engine.go

```diff
@@ -37,31 +38,33 @@ type AlluxioEngine struct {
 	runtimeType string
 	Log         logr.Logger
 	client.Client
-	// gracefulShutdownLimits is the limit for the system to forcibly clean up.
-	gracefulShutdownLimits int32
-	retryShutdown          int32
-	initImage              string
-	MetadataSyncDoneCh     chan MetadataSyncResult
-	runtimeInfo            base.RuntimeInfoInterface
-	UnitTest               bool
-	lastCacheHitStates     *cacheHitStates
+	// defaultGracefulShutdownLimits is the limit for the system to forcibly clean up.
+	defaultGracefulShutdownLimits       int32
+	defaultCleanCacheGracePeriodSeconds int32
```

Comment:
I think we can set the default value as a constant but not a variable in the engine

Reply:
Thanks for your suggestions.

Reply:
updated.